<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LancerDeClass</class>
 <widget class="QMainWindow" name="LancerDeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LancerDe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startDeButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer Dé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showValueDeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher valeur du dé</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="showTextDeValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showTotalScoreButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher score total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetTotalScoreButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remettre à Zéro le score total</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="showTextTotalScoreValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showNumberJet">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de jet</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="showTextNumberJet">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableHistorique">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showHistorique">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuchoisir">
    <property name="title">
     <string>choisir</string>
    </property>
    <addaction name="actionde"/>
    <addaction name="actiondehisto"/>
   </widget>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>fichier</string>
    </property>
   </widget>
   <addaction name="menufichier"/>
   <addaction name="menuchoisir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionde">
   <property name="text">
    <string>de</string>
   </property>
  </action>
  <action name="actiondehisto">
   <property name="text">
    <string>dehisto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LancerDe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startDeButton</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>startDeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showValueDeButton</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>showValueDeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTotalScoreButton</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>showTotalScoreButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetTotalScoreButton</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>resetTotalScoreButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actiondehisto</sender>
   <signal>triggered()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>dehistoMenuButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionde</sender>
   <signal>triggered()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>deMenuButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startDeButtonClicked()</slot>
  <slot>showValueDeButtonClicked()</slot>
  <slot>showTotalScoreButtonClicked()</slot>
  <slot>resetTotalScoreButtonClicked()</slot>
  <slot>dehistoMenuButtonClicked()</slot>
  <slot>deMenuButtonClicked()</slot>
 </slots>
</ui>
